WWTMVC5/WWTMVC5.csproj

```diff
@@ -458,6 +452,18 @@
       <DependentUpon>ResourceService.svc</DependentUpon>
     </Compile>
     <Compile Include="WebServices\ServiceBase.cs" />
+    <Content Include="WWT2\ClientVersion.txt" />
+    <Content Include="WWT2\Copy of updateurl.txt" />
+    <Content Include="WWT2\dataversion.txt" />
+    <Content Include="WWT2\EqClientVersion.txt" />
+    <Content Include="WWT2\Eqdataversion.txt" />
+    <Content Include="WWT2\ExcelAddinUpdateUrl.txt" />
+    <Content Include="WWT2\ExcelAddinVersion.txt" />
+    <Content Include="WWT2\Message.txt" />
+    <Content Include="WWT2\minver.txt" />
+    <Content Include="WWT2\updateurl.txt" />
+    <Content Include="WWT2\version.txt" />
+    <Content Include="WWT2\warnver.txt" />
```

Comment:
Was the intent to add these files to the repo? I don't see them added as part of this PR.

Reply:
They were already part of the repo, just not included as content in the project